packages/@magic-sdk/react-native-expo/package.json

```diff
@@ -41,7 +41,7 @@
     "react": "^16.13.1",
     "react-native": "^0.62.2",
     "react-native-safe-area-context": "^4.4.1",
-    "react-native-webview": "8.1.2"
```

Comment:
Since Expo "controls"[1] the web view version by SDK would it make sense to make `react-native-webview` a peer-dependency in the Expo library? Or at least using a minimum version only like `>= 11.26.0`

[1] https://docs.expo.dev/versions/latest/sdk/webview/#installation

Reply:
Hey @adamjuhasz, not sure I follow here as `react-native-webview` is [already listed as a peer-dependency](https://github.com/magiclabs/magic-js/pull/436/files/903c2bfc7b770ee7bed76a7a7b75f8b6f8e51585#diff-98143491e3ab92933e1de301c96ce9f118c1edac7c57c45a663397ef364b8613R51). Are you suggesting we bump the min version to `11.26.0`?